lib/game.rb

```diff
@@ -1,15 +1,48 @@
 # frozen_string_literal: true
 
-require_relative 'board'
+require_relative 'move'
 
 class Game
-  attr_accessor :board
+  attr_accessor :board, :players, :console, :game_over
 
-  def initialize
-    @board = Board.new
+  def initialize(board, players, console)
+    @board = board
+    @players = players
+    @console = console
+    @game_over = false
   end
 
   def play
-    puts board.draw
+    take_turn until game_over
+  end
+
+  private
+
+  def current_player
+    players.current_player
+  end
+
+  def take_turn
+    console.board(board)
+    move = Move.new(current_player, board, console)
```

Comment:
This `Move` class doesn't seem necessary; plus the game would be creating new instances every time a player makes a move. It could be beneficial to think about what certain objects _know_ about in a real-life scenario. What do you think a `Player` should know about in tic-tac-toe?